### PR TITLE
Openssl, pubsub client

### PR DESCRIPTION
**Pull request by tkmct**
OPENSSL

### CODE REVIEW

File: demo-hello-world/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -18,5 +18,6 @@
             </intent-filter>
         </activity>
     </application>
-
+    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (omuomugin) [inline demo-hello-world/src/main/AndroidManifest.xml]: [memo]
We should keep in mind that when we are ready to write documents of "How to use this library", we need to include "add theses permissions".

File: plasma_android_sdk/src/main/java/com/cryptoeconomicslab/plasma_android_sdk/pubsub/Client.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.cryptoeconomicslab.plasma_android_sdk.pubsub
```
Comment (omuomugin) [inline plasma_android_sdk/src/main/java/com/cryptoeconomicslab/plasma_android_sdk/pubsub/Client.kt]: [IMO]
I guess it's ok for now to name this feature `pubsub` but actually it is network feature so I want it to be named `network` in sometime. (or if it is easy to fix I would like it to be fixed.)

Comment (tkmct) [reply plasma_android_sdk/src/main/java/com/cryptoeconomicslab/plasma_android_sdk/pubsub/Client.kt]: Right. I'm going to fix them in the near future. Please forgive me to merge this without fixing.

**Comment by tkmct** [comment]
ignoring lib seems to be a good idea but it might be annoying to add OpenSSL prebuild to use this library. I'm not sure if it's a good idea to force library user to do this thing.

**Comment by omuomugin** [comment]
> OpenSSL prebuild to use this library.

Oh, ok I thought those files are auto generated by building rust. 

If not I think we should have some kind of comment to explain how it works.

**Comment by tkmct** [comment]
Definetely

**Pull request merged**